hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -1034,6 +1035,24 @@ public function upgrade_1025() {
   }
 
   /**
+   * Alters civicrm_hrjobcontract_hour table to make default values for fte_num
+   * and fte_denom equal to 0.
+   * 
+   * @return boolean
+   *   True on success
+   * @todo Rename upgrade!!  To upgrade_1026()  Include in upgrade bundle!
```

Comment:
It was a note for me, I had to use a function name different to upgrade_1026, since I had already used that name for another upgrade, on a different branch.  Sorry about that!